docs/en/guides/yolo-data-augmentation.md

```diff
@@ -0,0 +1,310 @@
+| <img src="https://github.com/ultralytics/docs/releases/download/0/augmentation_identity.avif" alt="augmentation_identity" width="85%"/> | <img src="https://github.com/ultralytics/docs/releases/download/0/augmentation_erasing_ex1.avif" alt="erasing_ex1_augmentation" width="85%"/> | <img src="https://github.com/ultralytics/docs/releases/download/0/augmentation_erasing_ex2.avif" alt="erasing_ex2_augmentation" width="85%"/> | <img src="https://github.com/ultralytics/docs/releases/download/0/augmentation_erasing_ex3.avif" alt="erasing_ex3_augmentation" width="85%"/> |
+
+## FAQ 🙋‍♂️
+
+### There are too many augmentations to choose from. How do I know which ones to use?
+
+Choosing the right augmentations depends on your specific use case and dataset. Here are a few general guidelines to help you decide:
+
+- In most cases, slight variations in color and brightness are beneficial. The default values for `hsv_h`, `hsv_s`, and `hsv_v` are a solid starting point.
+- If the camera's point of view is consistent and won't change once the model is deployed, you can likely skip geometric transformations such as `rotation`, `translation`, `scale`, `shear`, or `perspective`. However, if the camera angle may vary and you need the model to be more robust, it's better to keep these augmentations.
+- Use the `mosaic` augmentation only if having partially occluded objects or multiple objects per image is acceptable and does not change the label value. Alternatively, you can keep `mosaic` active but increase the `close_mosaic` value to disable it earlier in the training process.
+
+In short: keep it simple. Start with a small set of augmentations and gradually add more as needed. The goal is to improve the model's generalization and robustness, not to overcomplicate the training process. Also, make sure the augmentations you apply reflect the same data distribution your model will encounter in production.
+
+### When starting a training, a see a `albumentations: Blur[...]` reference. Does that mean the YOLO package runs additional augmentation like blurring?
+
+If the `albumentations` package is installed, Ultralytics automatically applies a set of extra image augmentations using it. These augmentations are handled internally and require no additional configuration.
+
+You can find the full list of applied transformations in our [technical documentation](https://docs.ultralytics.com/reference/data/augment/#ultralytics.data.augment.Albumentations). Note that only augmentations with a probability (`p`) greater than `0` are active. These are purposefully applied at low frequencies to mimic real-world visual artifacts, such as blur or grayscale effects.
+
+### When starting a training, I don't see any reference to albumentations. Why?
+
+Check if the `albumentations` package is installed. If not, you can install it by running `pip install albumentations`. Once installed, the package should be automatically detected and used by Ultralytics.
+
+## Example Configuration
```

Comment:
@picsalex let's move this up before the FAQs. Our current structure is to have FAQs as the last H2 section on each page. Maybe these could be near the top of the page under an `## Example Training with Augmentations` section?